mmpose/models/detectors/multiview_pose.py

```diff
@@ -153,34 +118,23 @@ def get_voxel(self, heatmaps, meta, grid_size, grid_center, cube_size):
         cubes[cubes != cubes] = 0.0
         cubes = cubes.clamp(0.0, 1.0)
 
-        cubes = cubes.view(batch_size, num_joints, cube_size[0], cube_size[1],
-                           cube_size[2])
+        cubes = cubes.view(batch_size, num_channels, cube_size[0],
+                           cube_size[1], cube_size[2])
         return cubes, grids
 
-    def forward(self, heatmaps, meta, grid_size, grid_center, cube_size):
-        cubes, grids = self.get_voxel(heatmaps, meta, grid_size, grid_center,
-                                      cube_size)
+    def forward(self, feature_maps, meta, grid_size, grid_center, cube_size):
+        cubes, grids = self.get_voxel(feature_maps, meta, grid_size,
+                                      grid_center, cube_size)
         return cubes, grids
 
 
 @POSENETS.register_module()
-class VoxelPose(BasePose):
+class DetectAndRegress(BasePose):
     """VoxelPose Please refer to the `paper <https://arxiv.org/abs/2004.06239>`
```

Comment:
These comments need to be more general. For example, this detector is for the `detect-and-regress setting`, in which we support `VoxelPose` for now.